test/main.cpp

```diff
@@ -357,6 +368,17 @@ EXPORT int main(int argc, char const* argv[])
 		if (filter && tests_to_run.count(_g_unit_tests[i].name) == 0)
 			continue;
 
+		std::snprintf(dir, sizeof(dir), "sub_%i", i);
+		std::string unit_dir = combine_path(current_working_directory(), dir);
```

Comment:
in fact, since this is in master (and you have C++11) you could just do:

auto unit_dir = combine_path(dir, "sub_" + std::to_string(i));